molgenis-mail/src/main/java/org/molgenis/mail/MailSettings.java

```diff
@@ -0,0 +1,20 @@
+package org.molgenis.mail;
+
+interface MailSettings
+{
+	String getHost();
```

Comment:
Are we missing out on settings that would require configuring a sendmail server on CentOS?